libs/gnutls/Makefile

```diff
@@ -109,19 +109,28 @@ endef
 CONFIGURE_ARGS+= \
 	--enable-shared \
 	--enable-static \
+	--disable-bash-tests \
+	--disable-doc \
+	--disable-full-test-suite \
+	--disable-gcc-warnings \
+	--disable-gtk-doc-html \
```

Comment:
this is disabled by default, why added?

Reply:
gtk-doc is disabled by default but gtk-doc-html is according to the configure script. Although it's probably dependent on the former...

Reply:
Unless there was a specific issue let's undo that. This is disabled by default.